File: src/main/java/com/vacaciones/payLoads/CreateUserPayload.java
package com.vacaciones.payLoads;

import java.sql.Date;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
public class CreateUserPayload {

    private String document;
    private String name;
    private String lastName;
    private String email;
    private String password;
    private Integer holyDays;
    private String position;
    private String userType;
    private String location;
    private Date singInDate;
    private Date withdrawalDate;

}

File: src/main/frontend/src/apiCall/payloads/CreateUserPayload.js
export default class CreateUserPayload{
    document
    name
    lastName
    email
    password
    holyDays
    position
    userType
    location
    singInDate
    widthdrawalDate

    constructor (document, name, lastName, email, password, holyDays, position,
                 userType, location, singInDate, widthdrawalDate){
                    this.document = document
                    this.name = name
                    this.lastName = lastName
                    this.email = email
                    this.password = password
                    this.holyDays = holyDays
                    this.position = position
                    this.userType = userType
                    this.location = location
                    this.singInDate = singInDate
                    this.widthdrawalDate = widthdrawalDate
                 }
}